apps/leo_storage/src/leo_storage_statistics.erl

```diff
@@ -104,21 +104,31 @@ get_and_set_mq_value([]) ->
 get_and_set_mq_value([{?SNMP_MQ_NUM_OF_DEL_DIR = Id, ?QUEUE_ID_DEL_DIR}|Rest]) ->
     V = lists:foldl(
           fun(QId, SoFar) ->
-                  N = case catch leo_mq_api:status(QId) of
+                  RetN = case catch leo_mq_api:status(QId) of
                           {ok, Ret} ->
-                              leo_misc:get_value(
-                                ?MQ_CNS_PROP_NUM_OF_MSGS, Ret, 0);
+                              case leo_misc:get_value(
+                                     ?MQ_CNS_PROP_NUM_OF_MSGS, Ret, 0) of
+                                  N when is_number(N) ->
+                                      N;
+                                  _ ->
+                                      0
+                              end;
```

Comment:
What I meant in the previous comment is that the value should be rounded to leo_math:power(2,32) as check_number does because it can be higher than 4294967296.